README.md

```diff
@@ -1,81 +1,34 @@
+<div align="center">
+
 # `cyberdrop-dl-patched`
+*Bulk asynchronous downloader for multiple file hosts*
 
-**Bulk downloader for multiple file hosts**
+![PyPI - Version](https://img.shields.io/pypi/v/cyberdrop-dl-patched)
+![PyPI - Python Version](https://img.shields.io/pypi/pyversions/cyberdrop-dl-patched)
+![Docs](https://img.shields.io/badge/docs-wiki-blue?link=https%3A%2F%2Fscript-ware.gitbook.io%2Fcyberdrop-dl)
+![GitHub License](https://img.shields.io/github/license/jbsparrow/CyberDropDownloader)
+![PyPI - Downloads](https://img.shields.io/pypi/dm/cyberdrop-dl-patched)
 
-[![PyPI version](https://badge.fury.io/py/cyberdrop-dl-patched.svg)](https://badge.fury.io/py/cyberdrop-dl-patched)
-[![Downloads](https://static.pepy.tech/badge/cyberdrop-dl-patched)](https://pepy.tech/project/cyberdrop-dl-patched)
-[![Downloads](https://static.pepy.tech/badge/cyberdrop-dl-patched/month)](https://pepy.tech/project/cyberdrop-dl-patched)
-[![Downloads](https://static.pepy.tech/badge/cyberdrop-dl-patched/week)](https://pepy.tech/project/cyberdrop-dl-patched)
+[![Discord](https://discordapp.com/api/guilds/1070206871564197908/widget.png?style=banner2)](https://discord.com/invite/kbZCxz22Qp)
```

Comment:
@jbsparrow this is the original discord invite link from Jules. Clicking it shows `<deleted user...> invited you`. Not really that important but it will be nice if you update it with an invite link from you.